packages/light-client/README.md

```diff
@@ -80,11 +80,11 @@ await lightclient.start();
 logger.info("Lightclient synced");
 
 lightclient.emitter.on(LightclientEvent.lightClientFinalityHeader, async (finalityUpdate) => {
-    logger.info(finalityUpdate);
+  logger.info(finalityUpdate);
 });
 
 lightclient.emitter.on(LightclientEvent.lightClientOptimisticHeader, async (optimisticUpdate) => {
-    logger.info(optimisticUpdate);
```

Comment:
How did you find all these formatting issues, just manual review?

Reply:
Those were identified by `yarn docs:lint`

Reply:
This line actually fixes the format directly without failing, then the workflow proceeds.

Reply:
> This line actually fixes the format directly without failing, then the workflow proceeds.

this seems wrong, at least in `.github/workflows/docs-check.yml` it doesn't seem right ot run lint:fix, should just do lint check instead

Reply:
> Makes sense, changed so that it runs `yarn docs:lint`

Would be good to double check that it now also fails if there is a lint issue. Ideally, we should not have this change in this PR

Reply:
It does, I had to [fix](https://github.com/ChainSafe/lodestar/pull/6566/commits/6a0f10f7529958dcaebe96c0b336809501aa2bb6) an extra lint issue :)